### ISSUE TITLE
Allow supressing query frontend log outputs

### ISSUE THREAD

**Issue by hhamalai** [grafana/loki#4014]
Query frontend produces a huge amount of log outputs from its statistics, like
```
ts=2021-07-20T09:06:27.916892789Z caller=spanlogger.go:53 org_id=xxx traceID=ee4d974a7f383c1 Summary.BytesProcessedPerSecond="0 B" Summary.LinesProcessedPerSecond=0 Summary.TotalBytesProcessed="0 B" Summary.TotalLinesProcessed=0 Summary.ExecTime=9.633976ms
ts=2021-07-20T09:06:27.933565383Z caller=spanlogger.go:53 org_id=xxx traceID=622beb1fdbda680 Ingester.TotalReached=6 Ingester.TotalChunksMatched=0 Ingester.TotalBatches=0 Ingester.TotalLinesSent=0 Ingester.HeadChunkBytes="0 B" Ingester.HeadChunkLines=0 Ingester.DecompressedBytes="0 B" Ingester.DecompressedLines=0 Ingester.CompressedBytes="0 B" Ingester.TotalDuplicates=0 Store.TotalChunksRef=0 Store.TotalChunksDownloaded=0 Store.ChunksDownloadTime=0s Store.HeadChunkBytes="0 B" Store.HeadChunkLines=0 Store.DecompressedBytes="0 B" Store.DecompressedLines=0 Store.CompressedBytes="0 B" Store.TotalDuplicates=0
```

It would be nice to be able to suppress these lines. At the moment, there exists no such configuration option I'm aware of.

**Comment by cyriltovena** [comment]
We need to wrap this 

https://github.com/grafana/loki/blob/main/pkg/logqlmodel/stats/context.go#L45-L82

with `level.Info(log).Log()....`  so that it can be removed.

**Comment by magcode** [comment]
Plus one, it really spams my syslog.

**Comment by cyriltovena** [comment]
Closed by #4872
